src/AssociatedContentAssignment.js

```diff
@@ -42,9 +48,14 @@ export default class AssociatedContentAssignment extends Component {
       );
     const descriptionHtml = doc.querySelector("body").innerHTML;
 
+    const externalTool = this.settings.querySelector(
```

Comment:
So here instead of `this.settings.querySelector(...)` it would be `this.state. externalTool`

Reply:
got it, thanks